pilot/pkg/features/pilot.go

```diff
@@ -704,6 +704,10 @@ var (
 
 	StackdriverAuditLog = env.Register("STACKDRIVER_AUDIT_LOG", false, ""+
 		"If enabled, StackDriver audit logging will be enabled.").Get()
+
+	PersistOldestWinsHeuristicForWildcardHostMatching = env.Register("PERSIST_OLDEST_FIRST_HEURISTIC_FOR_WILDCARD_HOST_MATCHING", false,
```

Comment:
@keithmattix can you document what's the behavior change with `PERSIST_OLDEST_FIRST_HEURISTIC_FOR_WILDCARD_HOST_MATCHING = false`

And also PERSIST_OLDEST_FIRST_HEURISTIC_FOR_WILDCARD_HOST_MATCHING = true.

I feel both would change something

Reply:
Yes, is a change in behavior regardless of the flag's value because the current behavior is broken; overlapping wildcards result in duplicate virtual hosts. The default behavior (flag off) matches services to virtual services configuration based on the most specifc VS host. With the flag set to true, the oldest matching virtual service is selected when building the virtual host configuration for the service.

Reply:
Fair enough. Will rename to PERSIST_OLDEST_FIRST_HEURISTIC_FOR_VIRTUAL_SERVICE_HOST_MATCHING